src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -140,15 +142,16 @@ const WorkspaceContainer = ({ namespace, name, breadcrumbs, topBarContent, title
           div({ style: { flex: 1 } }, [
             children
           ]),
-          div({ style: Style.elements.contextBarContainer }, [
-            h(ContextBarButtons, {
-              setDeletingWorkspace,
-              setCloningWorkspace,
-              setSharingWorkspace,
-              isOwner,
-              canShare
-            })
-          ])
+          workspace && h(ContextBar, {
```

Comment:
removed the styling from the workspace container and put it in the context bar itself